src/components/calculator.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import NumericKeys from './numericKeys';
+import OperatorKeys from './operatorKeys';
+import OutputScreen from './outputScreen';
+
+class Calculator extends Component {
+  constructor() {
+    super();
+    this.state = { };
+  }
+
+  render() {
+    return (
+      <div className="calculator">
+        <OutputScreen />
+        <OperatorKeys />
+        <NumericKeys />
+      </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update all your **components files** to start with a capital letter for React conventions and best practices. e.g.: `Calculator.jsx` instead of `calculator.jsx`,... 
I suggest you apply this approach to all the components. I know you can do it :muscle: :muscle: